src/NServiceBus.Testing/Handler.cs

```diff
@@ -105,16 +106,16 @@ public Handler<T> ExpectNotSendLocal<TMessage>(Func<TMessage, bool> check)
         /// </summary>
         public Handler<T> ExpectReturn<TEnum>(Func<TEnum, bool> check)
```

Comment:
This is from the callbacks API. There was a `Return` and a `Reply` which now is merged in `Reply`